### PR TITLE
Claudia 6159 dockerhub

### PR DESCRIPTION
**Pull request by hmunfru**
#### REVIEWERS

@jesuspg 
#### DESCRIPTION

Refactoring skuld for dockerhub
#### TESTING

Locally

### CODE REVIEW

File: docker/AcceptanceTests/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# How to use Skuld with Docker
+
+There are several options to use Skuld very easily using docker. These are (in order of complexity):
+
+- _"Check the unit tests associated to the component"_. See Section **1. Run Unit Test of Skuld**.
+- _"Check the acceptance tests are running properly"_ or _"I want to check that my Skuld instance run properly"_ . See Section **2. Run Acceptance tests**.
+
+You do not need to do all of them, to check if your Skuld instance run properly. You do need to have docker in your machine. See the [documentation](https://docs.docker.com/installation/) on how to do this. Additionally, you can use the proper FIWARE Lab docker functionality to deploy dockers image there. See the [documentation](https://docs.docker.com/installation/).
+
+Docker allows you to deploy an Skuld container in a few minutes. This method requires that you have installed docker or can deploy container into the FIWARE Lab (see previous details about it). Consider this method if you want to try Skuld and do not want to bother about losing data. Follow these steps:
```
Comment (jesuspg) [inline docker/AcceptanceTests/README.md:3]: please replace "an" by "a"

**Comment by flopezag** [comment]
docker/README.md is not the same that docker/AcceptanceTests/README.md?

File: docker/AcceptanceTests/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    export KEYSTONE_IP=<IP of the keystone instance>
+    export ADM_TENANT_ID=<admin tenant id in the OpenStack environment>
+    export ADM_TENANT_NAME=<admin tenant name>
+    export ADM_USERNAME=<admin username>
+    export ADM_PASSWORD=<admin password>
+    export Region1=<Region name>
+    export OS_USER_DOMAIN_NAME=<OpenStack user domain name>
+    export OS_PROJECT_DOMAIN_NAME=<OpenStack project domain name>	
+
+Take it, You should move to the AcceptanceTests folder `./AcceptanceTests`. Just create a new docker image executing `docker build -t fiware-skuld-acceptance .`. To see that the image is created run `docker images` and you see something like this:
```
Comment (jesuspg) [inline docker/AcceptanceTests/README.md]: please, remove ./ from folder name

Comment (jesuspg) [inline docker/AcceptanceTests/README.md]: please, remove ./ from folder name

**Comment by hmunfru** [comment]
Yes docker/README.md is the same that docker/AcceptanceTests/README.md. My problem is that I create the docker image in dockerhub https://hub.docker.com/r/fiware/fiware-skuld/ based on the Dockerfile in AcceptanceTests. If I don't add the README in the folder AcceptanceTests there is not information for the image. Dockerhub is taking the README for the general, instead of the one from docker folder.

**Comment by jesuspg** [comment]
LGTM

**Pull request merged**